### PR TITLE
Fix issues with storyboards and segues

### PR DESCRIPTION
**Pull request by gcasa**
The purpose of this issue is to fix a number of remaining issues with NSPopover, principally the positioning issue and some issues with NSStoryboardSegue.

### CODE REVIEW

File: Source/NSPopover.m [hunk 1]
```diff
@@ -249,10 +249,12 @@ - (void) showRelativeToRect: (NSRect)positioningRect
   NSRect screenRect;
   NSRect windowFrame;
   NSRect viewFrame;
-
+  NSWindow *window = nil;
+  
   [_contentViewController loadView];
   view = [_contentViewController view];
-  viewFrame = [view frame];
+  window = [view window];
```
Comment (fredkiefer) [inline Source/NSPopover.m:256]: I don't see where this new variable gets used.

File: Source/NSStoryboardSegue.m [hunk 2]
```diff
@@ -174,24 +174,23 @@ - (void) perform
...
-	  NSPopover *po = [[NSPopover alloc] init];
 	  NSRect rect = [_popoverAnchorView frame];
 
-	  _popover = po; // weak... since we manually release...
-	  [po setBehavior: _popoverBehavior];
-	  [po setContentViewController: _destinationController];
-	  [po showRelativeToRect: rect
-			  ofView: _popoverAnchorView
-		   preferredEdge: _preferredEdge];
+	  NSLog(@"anchor view = %@", _popoverAnchorView);
```
Comment (fredkiefer) [inline Source/NSStoryboardSegue.m:179]: Should rather be a NSDebugLLog

File: Source/NSPopover.m (hunk 1 above)
Comment (gcasa) [reply Source/NSPopover.m:256]: Right now things are in flux a bit.   I will be moving a lot of things to NSDebugLog or removing them once I am done.

File: Source/NSPopover.m [hunk 3]
```diff
@@ -294,8 +295,8 @@ - (void) showRelativeToRect: (NSRect)positioningRect
   [_realPanel setFrame: windowFrame display: YES];
   [_realPanel makeKeyAndOrderFront:self];
 
-  NSDebugLog(@"Showing relative to in window %@",NSStringFromRect(positioningRect));
-  NSDebugLog(@"Showing relative to in screen %@",NSStringFromRect(screenRect));
+  NSLog(@"Showing relative to in window %@",NSStringFromRect(positioningRect));
```
[the remark this answers is not in the document]
Comment (gcasa) [reply Source/NSPopover.m:298]: No, I will be taking it out before I finish.

**Comment by gcasa** [comment]
@fredkiefer I am working on some of the issues with segues as we had discussed months ago.  The code that does the action for the segue is not generating (or decoding) that object properly so some of the data is not present.   I am going through some of the logic and fixing issues.

**Comment by gcasa** [comment]
OBE by refactor.  No longer needed, so... closing.

**Pull request closed without merging**